sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/model/ErrorReport.java

```diff
@@ -112,7 +112,7 @@ public Throwable getThrowable() {
     /**
      * Sets the throwable.
      *
-     * @deprecated This method has been deprecated, use {@link #setStackTrace(String)} instead.
+     * @deprecated This method has been deprecated.
      * @param throwable A throwable to set.
      */
     @Deprecated
```

Comment:
So the comment was to remove the `setThrowable` method altogether, it's not part of the supported API surface. The public keyword was a side effect of not having the internal keyword in Java.